rql/query/src/test/java/org/eclipse/ditto/rql/query/things/ThingPredicatePredicateVisitorTest.java

```diff
@@ -137,6 +137,13 @@ public void matchingStringLike() {
                 .isTrue();
     }
 
+    @Test
+    public void matchingStringILike() {
+        // the sut already works on regex Pattern - the translation from "*" to ".*" followed by case insensitivity is done in LikePredicateImpl
+        doTest(sut.visitLike("this-is.*"), JsonValue.of("THIS-IS-THE-CONTENT"))
```

Comment:
`visitLike`?
You probably want to test visitILike, or not?
The comment is copy&paste from the like test and wrong.